tests/karma/single.conf.js

```diff
@@ -48,7 +48,11 @@ module.exports = function (config) {
 			{pattern: "tests/js/**/*.html", included: false, served: true, watch: true},
 			{pattern: "src/js/**/*.js", included: false, served: true, watch: true},
 			{pattern: "dist/**/*", included: false, served: true, watch: true},
-			"tests/karma/single.test.path.js",
+			{pattern: "libs/BezierCurve.js", included: false, served: true},
```

Comment:
Instead of modify this file we can update `all.conf.js`

Reply:
done